swarm/api/http/server.go

```diff
@@ -626,9 +626,9 @@ func (s *Server) HandleGet(w http.ResponseWriter, r *Request) {
 	log.Debug("handle.get", "ruid", r.ruid, "uri", r.uri)
 	getCount.Inc(1)
 	var err error
-	key := r.uri.Key()
-	if key == nil {
-		key, err = s.api.Resolve(r.uri)
+	addr := r.uri.Key()
```

Comment:
Should `URI.Key()` be renamed to `URI.Address()`? BTW, URI type already contains `addr` and `Addr`, but fortunately with nice comments.